spec/controllers/api/public/v1/providers/courses_controller_spec.rb

```diff
@@ -98,4 +98,74 @@
       end
     end
   end
+
+  describe "#show" do
+    context "when course exists" do
+      let!(:course) { create(:course, provider: provider) }
+
+      it "returns the course" do
+        get :show, params: {
```

Comment:
can we move these get requests into a before block so it sets up an arrange/act/assert structure

Reply:
Done